src/cookies.ts

```diff
@@ -2,6 +2,12 @@
 import Cookies from 'js-cookie';
 import { GuCookie, GuPurposeState } from './types';
 
+declare global {
+    interface Window {
+        guardian?: { config?: { page?: { isPreview?: boolean } } };
+    }
+}
+
```

Comment:
Are we using this type declaration in anyway? Is `window.guardian.config.page.isPreview` something we can use to fix this bug? Or was that something you'd attempted?

Reply:
No. That was from something else I was trying and forgot to remove.